files/ja/mdn/structures/page_types/api_method_subpage_template/index.html

```diff
@@ -26,7 +26,7 @@ <h3 id="Top_macros" name="Top_macros">先頭で使用するマクロ</h3>
 
 <ul>
  <li>\{{APIRef("<em>GroupDataName</em>")}} — this generates the left hand reference sidebar showing quick reference links related to the current page. For example, every page in the <a href="/ja/docs/Web/API/WebVR_API">WebVR API</a> has the same sidebar, which points to the other pages in the API. To generate the correct sidebar for your API, you need to add a GroupData entry to our KumaScript GitHub repo, and include the entry's name inside the macro call in place of <em>GroupDataName</em>. See our <a href="/ja/docs/MDN/Contribute/Structures/API_references/API_reference_sidebars">API reference sidebars</a> guide for information on how to do this.</li>
- <li>\{{Draft}} — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
+ <li>\ — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
```

Comment:
This is a description of the `{{Draft}}` macro, so removing the `{{Draft}}` breaks it.
If the `{{Draft}}` macro shouldn't be used, it is necessary to remove entire the paragraph, I think.

Reply:
👋 @mfuji09 

Thank you for the review. I will address the 3 issues you mentioned.